vent/api/plugin_helpers.py

```diff
@@ -38,10 +38,12 @@ def get_path(self, repo, core=False):
         if repo.endswith(".git"):
             repo = repo.split(".git")[0]
         org, name = repo.split("/")[-2:]
-        if core:
-            path = join(self.path_dirs.base_dir, '.internals/plugins/')
-        else:
-            path = self.path_dirs.plugins_dir
+        path = self.path_dirs.plugins_dir
+        # commenting out in case this may cause bugs
```

Comment:
this appears to be working as expected, so the commented out code can be removed.